src/model/test/Space_GTest.cpp

```diff
@@ -79,6 +79,44 @@
 using namespace openstudio;
 using namespace openstudio::model;
 
+struct SurfaceAreaCounter
```

Comment:
this is for -Wunused-but-set-variable on apple clang 13.1.6, but I also used the occasion to dry up the code